apiserver/facades/client/modelupgrader/upgrader_test.go

```diff
@@ -167,11 +167,28 @@ func (s *modelManagerUpgradeSuite) assertUpgradeModelForControllerModelJuju3(c *
 		ctrlState.EXPECT().MachineCountForSeries(
 			"win2008r2", "win2012", "win2012", "win2012hv", "win2012hvr2", "win2012r2", "win2012r2",
```

Comment:
I guess this test should be failed.
Because there are some duplications(`win2012` for example) have been removed in 2.9

Reply:
fixed dupes